src/components/Footer/LicenseText.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="h-max self-center">
+    <p class="copyright max-w-100 footer-text">
+      © <span>2018-{{ new Date().getFullYear() }}</span
+      ><contributorsText />
+    </p>
+    <p class="copyright max-w-100 footer-text"><licenseText /></p>
+  </div>
+</template>
+
+<script setup lang="tsx">
+const WITNET_FOUNDATION_URL = 'https://witnet.foundation'
+const CREATIVE_COMMONS_URL = 'https://creativecommons.org/publicdomain/zero/1.0/'
+const contributorsText = () => (
+  <span>
+    {' '}
+    by{' '}
+    <a href={WITNET_FOUNDATION_URL} target="_blank">
+      Witnet Foundation
+    </a>{' '}
+    and individual contributors.
+  </span>
+)
+const licenseText = () => (
+  <span>
+    {' '}
+    Content available under a{' '}
+    <a href={CREATIVE_COMMONS_URL} target="_blank">
+      Creative Commons License
+    </a>
+    .
+  </span>
+)
```

Comment:
We can create an slot for the contributors and the license and avoid using `jsx`